src/browser/BrowserAction.cpp

```diff
@@ -406,11 +406,11 @@ QString BrowserAction::getErrorMessage(const int errorCode) const
     case ERROR_KEEPASS_CANNOT_DECRYPT_MESSAGE:
         return QObject::tr("Cannot decrypt message");
     case ERROR_KEEPASS_TIMEOUT_OR_NOT_CONNECTED:
-        return QObject::tr("Timeout or cannot connect to KeePassXC");
+        return QObject::tr("Cannot connect to KeePassXC. Is Browser Integration enabled?");
     case ERROR_KEEPASS_ACTION_CANCELLED_OR_DENIED:
         return QObject::tr("Action cancelled or denied");
     case ERROR_KEEPASS_CANNOT_ENCRYPT_MESSAGE:
-        return QObject::tr("Cannot encrypt message or public key not found. Is Native Messaging enabled in KeePassXC?");
+        return QObject::tr("Message encryption failed. Is KeePassXC open and connected?");
```

Comment:
"Is KeePassXC running?" I don't think there is an obvious way to see if it is "connected" otherwise this error message wouldn't show up.